nautobot/dcim/tests/test_views.py

```diff
@@ -614,26 +614,74 @@ def test_powerports(self):
         # Create Power Port for device
         powerport1 = PowerPort.objects.create(device=devices[0], name="Power Port 11")
         powerfeed1 = PowerFeed.objects.create(
-            power_panel=self.powerpanels[0], name="Power Feed 11", phase="three-phase"
+            power_panel=self.powerpanels[0],
+            name="Power Feed 11",
+            phase="single-phase",
```

Comment:
I hope y'all don't mind me changing these to Single Phase. Three-phase adds legs, which I don't quite understand.

Reply:
No test is explicitly testing for three phase functionality so 👍